code/ai/aiturret.cpp

```diff
@@ -1605,8 +1605,9 @@ void turret_set_next_fire_timestamp(int weapon_num, weapon_info *wip, ship_subsy
 {
 	Assert(weapon_num < MAX_SHIP_WEAPONS);
 	float wait = 1000.0f;
+	int burst_shots = (wip->burst_flags[Weapon::Burst_Flags::Num_firepoints_burst_shots] ? turret->system_info->turret_num_firing_points - 1 : wip->burst_shots);
```

Comment:
Why is this the firing points - 1?

Reply:
This is to match the behavior of how burst shots is handled. If the number of burst shots is greater than the burst counter (which is incremented on each burst shot and reset when the last shot in the burst is fired) that means this shot is a burst shot, if they're equal, this is the end of a burst and it should use firewait. So 0 burst shots is as normal, every shot is firewait, 1, means a two shot burst, every other shot gets burst delay or firewait, and so on.

Essentially, for the same reason that `$Burst Shots` is decremented by 1 when it is parsed. A way of thinking about it is that the internally used `burst_shots` value is always "how many burst delay shots happen between each firewait shot"

Reply:
Ah ok.

Does it make sense to add a comment essentially saying "this is intentional; see parsing" or similar?

Reply:
`wip->burst_shots` doesn't mention this, so yeah, I'll add a comment clarifying how that works, and refer these spots to that comment